bitcoin/secp256k1_main.py

```diff
@@ -311,16 +302,18 @@ def ecdsa_raw_sign(msg,
         ndata = ffi.new("char [32]", usenonce)
         usenonce = (nf, ndata)
     if formsg:
-        sig = newpriv.ecdsa_sign_recoverable(msg, raw=rawmsg)
-        s, rid = newpriv.ecdsa_recoverable_serialize(sig)
+        if rawmsg:
+            sig = newpriv.sign_recoverable(msg, hasher=None)
+        else:
+            sig = newpriv.sign_recoverable(msg)
+        s, rid = sig[:64], bytes_to_int(sig[64:])
```

Comment:
As well as other testing, please do test whether generated signatures still verify against a Core node's verification (this was added in develop recently), thanks.